.github/interop/required.json

```diff
@@ -618,5 +618,48 @@
     "s2n-quic": [],
     "s2n-quic-rustls": [],
     "xquic": []
+  },
+  "resumption": {
+    "aioquic": [
+      "client",
+      "server"
+    ],
+    "kwik": [
+      "client",
+      "server"
+    ],
+    "lsquic": [
+      "client",
+      "server"
+    ],
+    "mvfst": [
+      "server"
+    ],
+    "neqo": [
+      "client",
+      "server"
+    ],
+    "ngtcp2": [
+      "client",
+      "server"
+    ],
+    "picoquic": [
+      "client",
+      "server"
+    ],
+    "quic-go": [
+      "client",
+      "server"
+    ],
+    "quiche": [
+      "client",
+      "server"
+    ],
+    "s2n-quic": [
+      "client",
+      "server"
+    ],
+    "s2n-quic-rustls": [],
+    "xquic": []
```

Comment:
Took out requirement for xquic and the mvfst client as it seems like we have an incompatibility with them.